src/KeyStreamCapture/KeyStreamCapture.csproj

```diff
@@ -11,7 +11,8 @@
     <RootNamespace>KeyStreamCapture</RootNamespace>
     <AssemblyName>KeyStreamCapture</AssemblyName>
     <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
-    <TargetFrameworkProfile>Client</TargetFrameworkProfile>
```

Comment:
Can we back out from this change? It seems unnecessary to the overall PR (and might impact the installer)